api/db/changelog/db.changelog-200-demographic-survey-v2-tweaks.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="dmohs" id="changelog-200-demographic-survey-v2-tweaks">
+    <sql>
```

Comment:
Should the two alter be in one liquibase file,i think we have been separating the sql in different files,  just in case we want to revert just one and not the other?

Reply:
199 adds five columns and has an alter table, so this seems consistent with the existing pattern to me.